utils/utils.jsx

```diff
@@ -253,6 +266,10 @@ export function extractFirstLink(text) {
     // strip out inline markdown images
     inText = inText.replace(/!\[[^\]]*]\([^)]*\)/g, '');
 
+    // remove markdown *, ~~ and _ characters
+    inText = loopReplacePattern(inText, /(\*|~~)(.*?)\1/, '$2');
+    inText = loopReplacePattern(inText, /([\s\n]|^)_(.*?)_([\s\n]|$)/, '$1$2');
```

Comment:
This doesn't work properly (I think), for example '_test_ _test2_' doesn't work, probably would be fixed with a $3 at the end of the replacement (we are removing any space after the underlined text).